test/factorial_test.js

```diff
@@ -0,0 +1,17 @@
+import { expect } from 'chai'
```

Comment:
make a test to see if when n is 0, it returns 1 since you made a test for negative numbers